test/mock/rtps/RTPSParticipantAttributes/fastdds/rtps/attributes/RTPSParticipantAttributes.hpp

```diff
@@ -638,4 +638,4 @@ class RTPSParticipantAttributes
 }  // namespace fastdds
 }  // namespace eprosima
 
-#endif  // FASTDDS_RTPS_ATTRIBUTES__RTPSPARTICIPANTATTRIBUTES_HPP
+#endif  // FASTDDS_RTPS_ATTRIBUTES__RTPSParticipantAttributes_HPP
```

Comment:
I would leave it as it was
```suggestion
#endif  // FASTDDS_RTPS_ATTRIBUTES__RTPSPARTICIPANTATTRIBUTES_HPP
```

Reply:
Same